### PR TITLE
testutil/network: Refactor network test utility to add CELES and UCELES to genesis

### PR DESCRIPTION
**Pull request by renaynay**
Refactor network test utility to add both CELES and UCELES to genesis.

### CODE REVIEW

**Review by evan-forbes — COMMENTED** [review]
I'm tempted to merge this since its blocking, but we will change it very soon after without additional changes

File: testutil/network/network.go
Comment (evan-forbes) [reply testutil/network/network.go:124]: I think that DisplayDenom was meant to be a reference for any exchanges, but not actually used in code, but I could be wrong. 

@marbar3778 do you think we should keep this as is or use lower case?

File: testutil/network/network.go
Comment (evan-forbes) [inline testutil/network/network.go:124]: we have the correct token denomination, so we don't have to hold up this PR, but we might also want to get rid of giving CELES tokens just to avoid any confusion for future readers.

approving without tho

**Review by evan-forbes — APPROVED** [review]
:+1:

File: app/app.go
Comment (marbar3778) [inline app/app.go:92]: why is this being added? celes is not a denom on chain, 1,000,000uceles = 1 celes

File: testutil/network/network.go
Comment (marbar3778) [inline testutil/network/network.go:125]: there are two entries for celes here? one upper case and one lower,  is that on purpose?

**Review by marbar3778 — COMMENTED** [review]
not sure I follow what this is aiming at solving, I think this will cause more confusion if used

**Review by evan-forbes — CHANGES_REQUESTED** [review]
To attempt to clear up any confusion, all caps CELES was being added to the default genesis as a macro denomination of uceles. I think I was confused by this, so this PR changed that to lower case, that why we added 
`TokenDenom = "celes"`
https://github.com/renaynay/celestia-app/blob/0d496d75394aa9b003d1da7a0a30e4e4c8fb4e22/app/app.go#L92

to use in place of `DisplayDenom`. Tbh, I'm not sure why `DisplayDenom` exists in the first place, it was added in https://github.com/celestiaorg/celestia-app/pull/196 but I missed its purpose there. 

Its clear now that when creating a genesis file for testing, we should just be using `uceles`, which is the last change needed before this PR can be merged imho

**Comment by renaynay** [comment]
@marbar3778 can you help here?

**Comment by marbar3778** [comment]
you can have it as lower case or upper case for the display denom, Uppercase is inline with some chains but others like hub use lowercase. Uppercase is a bit better imo since that is what the token ticker will be. 


Using uceles as the default for testing should be done otherwise tooling you may build will rely on Celes but 1 Celes is not one celes but a micro celes, unless your equivalent of Wei or uatom will be Celes.

File: testutil/network/network.go
Comment (marbar3778) [reply testutil/network/network.go:125]: right now, if you try to use celes or CELES for staking it won't work, you will only be able to transfer the token since the system relies on uceles

Comment (evan-forbes) [reply testutil/network/network.go:125]: yes, we should remove this before merging

File: testutil/network/network.go
Comment (evan-forbes) [reply testutil/network/network.go:124]: I still think that we should remove this addition of Display denom to the default genesis file before merging

File: app/app.go
Comment (evan-forbes) [reply app/app.go:92]: This was added to replace the usage of DisplayDenom in the default genesis afaiu

**Comment by evan-forbes** [comment]
> you can have it as lower case or upper case for the display denom, Uppercase is inline with some chains but others like hub use lowercase. Uppercase is a bit better imo since that is what the token ticker will be.

Using uceles as the default for testing should be done otherwise tooling you may build will rely on Celes but 1 Celes is not one celes but a micro celes, unless your equivalent of Wei or uatom will be Celes.

thanks for this clarification, I have no opinion on which we use, but lets pick one before merging

**Comment by liamsi** [comment]
I think this can be closed as we are switching to [#token](https://github.com/celestiaorg/celestia-app/issues/279)

**Pull request closed without merging**
